docs/tutorials/scaleway.md

```diff
@@ -2,7 +2,7 @@
 
 This tutorial describes how to setup ExternalDNS for usage within a Kubernetes cluster using Scaleway DNS.
 
-Make sure to use **>=0.7.3** version of ExternalDNS for this tutorial.
+Make sure to use **>=0.7.4** version of ExternalDNS for this tutorial.
```

Comment:
I'm sure you are aware that that `0.7.4` is not release yet. Do you think it would be better to update this to indicate that the scaleway provider is not available in a release version of external-dns? Then after the next release this could be updated?

Reply:
Yeah that would make sense. Do we have a release date for 0.7.4 ? If it's soon enough, we could leave this PR open and merge it once 0.7.4 is released?

Reply:
I'm not sure when 0.7.4 will be released.

Reply:
There's no schedule, likely end of the month given other commitments.